northstar/src/runtime/minijail.rs

```diff
@@ -42,35 +43,40 @@ use tokio::{
     select, task, time,
 };
 
+lazy_static! {
```

Comment:
`lazy_static` is handy but horrible regarding error handling as you can see with the `unwraps`.
This block also assumes that it's accessed in a sync block on  a tokio runtime.

The `fd` represents some kind of state for the minijail module and therefore I think we should maintain it via composition. Initializing the minijail logging lazily also has an impact on the startup time of the *first* container ever started which might fuzz test.